runtime/mainnet/src/genesis.rs

```diff
@@ -0,0 +1,610 @@
+	owner: AccountId,
+	is_sufficient: bool,
+	min_balance: Balance,
+	name: Vec<u8>,
+	symbol: Vec<u8>,
+	decimals: u8,
+}
+
+/// Returns a JSON blob representation of the built-in `RuntimeGenesisConfig` identified by `id`.
+pub(crate) fn get_preset(id: &PresetId) -> Option<Vec<u8>> {
+	let patch = match id.as_str() {
+		MAINNET_DEV => development_config(),
+		MAINNET_LOCAL => local_config(),
+		MAINNET => live_config(),
+		_ => return None,
+	};
+	Some(
+		to_string(&patch)
+			.expect("serialization to json is expected to work. qed.")
+			.into_bytes(),
+	)
+}
+
+/// Returns a list of identifiers for available builtin `RuntimeGenesisConfig` presets.
+pub(crate) fn presets() -> Vec<PresetId> {
+	PRESETS.map(PresetId::from).to_vec()
+}
+
+/// Configures a development chain running on a single node, using the `mainnet` runtime.
+fn development_config() -> Value {
+	let mut endowed_accounts = dev_accounts();
+	endowed_accounts.push(MaintenanceAccount::get());
+	endowed_accounts.push(PotId::get().into_account_truncating());
+	endowed_accounts.push(TreasuryAccount::get());
+
+	genesis(
+		// Initial collators.
+		Vec::from([
+			// Single collator for development chain
+			(Keyring::Alice.to_account_id(), Keyring::Alice.public().into()),
+		]),
+		endowed_accounts,
+		Keyring::Alice.to_account_id(),
+		PARA_ID,
+		// AssetId reserved for DOT from AH.
+		vec![GenesisAsset {
+			id: 0,
+			owner: Keyring::Alice.to_account_id(),
+			is_sufficient: false,
+			min_balance: ExistentialDeposit::get(),
+			name: "DOT".into(),
+			symbol: "DOT".into(),
+			decimals: 10,
+		}],
+		vec![
+			Keyring::Alice.to_account_id(),
+			Keyring::Bob.to_account_id(),
+			Keyring::Charlie.to_account_id(),
+			Keyring::Dave.to_account_id(),
+			Keyring::Eve.to_account_id(),
+		],
+	)
+}
+
+/// Configures a local chain running on multiple nodes for testing purposes, using the `mainnet`
+/// runtime.
+fn local_config() -> Value {
+	let mut endowed_accounts = dev_accounts();
+	endowed_accounts.push(MaintenanceAccount::get());
+	endowed_accounts.push(PotId::get().into_account_truncating());
+	endowed_accounts.push(SudoAddress::get());
+	endowed_accounts.push(TreasuryAccount::get());
+
+	genesis(
+		// Initial collators.
+		Vec::from([
+			// Multiple collators for local development chain.
+			(Keyring::Alice.to_account_id(), Keyring::Alice.public().into()),
+			(Keyring::Bob.to_account_id(), Keyring::Bob.public().into()),
+		]),
+		endowed_accounts,
+		// Like the multisig used for live config, but with dev accounts.
+		derive_multisig::<Runtime>(
+			vec![
+				Keyring::Alice.to_account_id(),
+				Keyring::Bob.to_account_id(),
+				Keyring::Charlie.to_account_id(),
+				Keyring::Dave.to_account_id(),
+			],
+			2,
+		),
+		PARA_ID,
+		// AssetId reserved for DOT from AH.
+		vec![GenesisAsset {
+			id: 0,
+			owner: SudoAddress::get(),
+			is_sufficient: false,
+			min_balance: ExistentialDeposit::get(),
+			name: "DOT".into(),
+			symbol: "DOT".into(),
+			decimals: 10,
+		}],
```

Comment:
Similarly, (for mainnet only) please remove or change the name from "DOT"

Reply:
[57f4d96](https://github.com/r0gue-io/pop-node/pull/473/commits/57f4d964842e71ee4bc1a70c7ca184d4677dd56a)